src/components/RestaurantList/index.tsx

```diff
@@ -1,22 +1,18 @@
-import React from 'react';
+import React, { useContext } from 'react';
```

Comment:
```suggestion
import { useContext } from 'react';
```

다른  파일도 모두 마찬가지인데, `React`가 이제 안쓰이니까 import 안써도 되지 않을까요?

Reply:
ㅠㅜㅜ 호다닥 해서 그런가 이런 부분들을 놓친 게 많네요..
잡아주셔서 감사합니다~!